README.md

```diff
@@ -330,13 +330,13 @@ Collector's philosophy is to structure your events based on your UI hierarchy. W
 
 </div>
 
-In order to support the app/view/zone hierarchy, the event schema is defined by the following keys:
+In order to support the app/view/element hierarchy, the event schema is defined by the following keys:
 
 ```ts
 interface Event {
   app: string; // The application name
   view: string; // The current "view". Can be overwritten
-  zone?: string; // The current "feature"/"organism", such as a login form. Can be overwritten
+  elementTree: string[]; // The current tree structure for "feature"/"organism", such as a login form. Can be incremented
```

Comment:
What do you mean by "incremented" here?

Reply:
I'll change this to be more extensive regarding adding/removing items to it